<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoSetting</class>
 <widget class="QWidget" name="VideoSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="videoResolutionBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Framerate:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="videoCodecBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Format :</string>
   </property>
  </widget>
  <widget class="QSlider" name="videoQualitySlider">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="containerFormatBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>67</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Resolution:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Quality:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Codec:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="videoFramerateBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="videoInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Video Input:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="videoBitRateBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Bitrate:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
